pytest-embedded-serial-esp/pytest_embedded_serial_esp/serial.py

```diff
@@ -45,6 +44,7 @@ class EspSerial(Serial):
     """
 
     ESPTOOL_DEFAULT_BAUDRATE = 921600
+    BASE_OPERATION = ['--after', 'hard_reset']
```

Comment:
Is this necessary? `--after hard_reset` is the default in esptool and doesn't need to be specified.

Reply:
Good to know 😄 I will remove it then

Reply:
I also have a question about `esptool.get_default_connected_device`. 
I've noticed that if I don't call it every time after running `esptool.main`, I encounter a problem. 
Could you please explain why this happens and if there is any way to avoid calling get_default_connected_device every time?"